flink-python/pyflink/table/table_config.py

```diff
@@ -119,7 +116,7 @@ def get_built_in_database_name(self):
         """
         return self._j_table_config.getBuiltInDatabaseName()
 
-    def _set_built_in_database_name(self, built_in_database_name):
+    def set_built_in_database_name(self, built_in_database_name):
```

Comment:
Just reminder that : we should do the refactor change in a new PR. :)

Reply:
This is Because `TableConfig` class has been refactored to old design in this PR. So I refactored these newly added setter method in this PR too.

Reply:
That makes sense to me (see the summary of my review #2) :)